### PR TITLE
Diagram showing ledgers within transition-frontier

### PR DESCRIPTION
**Pull request by bkase**
Diagram showing ledgers within transition-frontier to make it clear where the snarked database is vs the masks.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -3,3 +3,4 @@
 src/lib/snarky/src/camlsnark_c/libsnark-caml/* linguist-vendored
 *.png filter=lfs diff=lfs merge=lfs -text
 docs/res/transition_frontier_diagram.png filter=lfs diff=lfs merge=lfs -text
+docs/res/ledgers_in_frontier.png filter=lfs diff=lfs merge=lfs -text
```
Comment (psteckler) [inline .gitattributes]: Does the `*.png` line above cover this one?

If not, could this be `docs/res/*.png`, so we don't have to add entries for each image?

Comment (bkase) [reply .gitattributes]: I don't think it does, but I'm not sure. `git lfs track <file.png>` is what I execute and that command appends to `.gitattributes`

Comment (cmr) [reply .gitattributes]: It should be changed to `**/*.png` to cover everything, but it seems like that line should be removed in favor of `git lfs` managing it.

Comment (bkase) [reply .gitattributes]: git lfs adds line 4 as well

Comment (nholland94) [reply .gitattributes]: I mentioned this offline, but you actually don't even need `**/*.png`. Just the `*.png` line that's there is enough to apply recursively to all files. You never need to run `git lfs` commands when working with any `*.png` files with that rule in place. Just add, commit, and forget.

**Pull request merged**